### PR TITLE
[ATS-100] Cover GeoJson classes by tests.

### PR DESCRIPTION
**Pull request by robertkuraj**

### CODE REVIEW

**Review by QuintinWillison — APPROVED** [review]
I'm not sure about the naming of `isLessThanZeroThenNil()`, but otherwise looks sensible. I've not looked at the tests in detail, but out of interest have you had a look at coverage for those?

**Comment by QuintinWillison** [comment]
Looks like check failures are down to CocoaPods-related issue:

> [!] CDN: trunk URL couldn't be downloaded: https://cdn.jsdelivr.net/cocoa/Specs/1/4/1/Ably/1.0.2/Ably.podspec.json Response: 429 429: Too Many Requests

**Comment by QuintinWillison** [comment]
Hmm, that didn't work. Still failing. Perhaps I'll leave it an hour and then try again.

**Comment by robertkuraj** [comment]
> Hmm, that didn't work. Still failing. Perhaps I'll leave it an hour and then try again.

I've just re-runned the checks and both passed 🎉

File: Core/Tests/Sources/Model/GeoJSON/GeoJSONMessageCLLocationTests.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            XCTAssertEqual(message.properties.accuracySpeed, 7.0)
+            XCTAssertEqual(message.properties.accuracyBearing, 5.0)
+            XCTAssertEqual(message.properties.bearing, 4.0)
+        } else {
+            XCTAssertNil(message.properties.accuracySpeed)
+            XCTAssertNil(message.properties.accuracyBearing)
+            XCTAssertNil(message.properties.bearing)
+        }
+        
+
```
[the remark this answers is not in the document]
Comment (robertkuraj) [reply Core/Tests/Sources/Model/GeoJSON/GeoJSONMessageCLLocationTests.swift]: Removed in PR update.

**Comment by robertkuraj** [comment]
> I'm not sure about the naming of `isLessThanZeroThenNil()`, but otherwise looks sensible. I've not looked at the tests in detail, but out of interest have you had a look at coverage for those?

It is the private extension method so I was not able to test it directly but I covered by the tests all the cases where this method is being used and affects the received results.

**Pull request merged**